### DISCUSSION TITLE
show IP address using Reverse Tunnel

### DISCUSSION THREAD

**Discussion by evrynet1** [gravitational/teleport#38781] [Ideas]
While using reverse tunnels only the text "tunnel" is being shown under Address column in GUI. It would be great to show the IP address of the node as with direct connections.

<img width="91" alt="Screenshot 2024-02-29 at 10 02 00" src="https://github.com/gravitational/teleport/assets/8157986/ec1df895-035a-458f-b199-1639c1785cf3">

**Comment by stevenGravy** [comment]
hi @evrynet1 , reverse tunnels configurations do not connect to Teleport services via the machine's IPs and ports. Since Teleport is using the reverse tunnel to maintain connections there is no direct connection IP to show that is in use.  You could optionally add IPs to the labels via static or dynamic labels.

**Comment by webvictim** [comment]
You can also see the IP address a node is using for its tunnel in the Teleport proxy logs - it may only be logged at `DEBUG` level, however.
